src/libs/ValidationUtils.js

```diff
@@ -285,12 +286,15 @@ function validateIdentity(identity) {
  * @returns {Boolean}
  */
 function isValidUSPhone(phoneNumber = '', isCountryCodeOptional) {
```

Comment:
To be consist with the recent change, I think it would be better if we benefit from the second param of `parsePhoneNumber`.

```js
function isValidUSPhone(phoneNumber = '', isCountryCodeOptional) {
    const phone = phoneNumber || '';
    const regionCode = isCountryCodeOptional ? CONST.COUNTRY.US : null;

    const parsedPhoneNumber = parsePhoneNumber(phone, {regionCode});
    return parsedPhoneNumber.possible && parsedPhoneNumber.regionCode === CONST.COUNTRY.US;
}

```

Reply:
Seems unrelated to this PR. Let this be handled somewhere else.

Reply:
> We only allow US phone numbers, right?

That is not technically true. There are some others that are accepted, however currently we are not releasing a public list of them for some internal reasons. We should be opening up to all countries again shortly 😄